.github/workflows/markdowncheck.yml

```diff
@@ -1,6 +1,13 @@
 name: Check Markdown links
 
-on: push
+on: 
+  push:
+    paths:
+      - '**.md' # only run when there are changes to markdown files in a push to any branch
```

Comment:
We rarely modify the markdown files, and sometimes the markdown linter lets us know that links which used to be correct have gone away.  Something to consider.

Reply:
Yeah, that's why I added the "sanity check" a couple of lines farther down - we'll run markdown check on every PR, but not necessarily every push.

Reply:
Actually, I think a better way would be to change the workflow to run anytime a .md file is modified and then as a scheduled nightly check.